docs/core/rid-catalog.md

```diff
@@ -160,10 +160,11 @@ The following list shows the most common RIDs used for each OS. It doesn't cover
 
 ## OS X RIDs
 
-- `osx-x64` (.NET Core 2.0 or later versions)
+- `osx-x64` (.NET Core 2.0 or later versions, min version is `osx.10.12-x64`)
```

Comment:
min -> minimum 
- or -
maybe we should just link to the prerequisites topics for each of these sections like this one:
https://github.com/dotnet/docs/blob/master/docs/core/macos-prerequisites.md